test/test_nci_workspace_cleaner.rb

```diff
@@ -47,7 +47,11 @@ def mkdir(path, mtime)
 
   def test_clean
     datetime_now = DateTime.now
-    mkdir('mgmt_6_days_old', datetime_now - 6)
+    mkdir("mgmt_#{NCI.current_series}_5_days_old", datetime_now - 5)
+    mkdir("mgmt_cnf_#{NCI.current_series}_6_days_old", datetime_now - 6)
+    mkdir("mgmt_cnf_#{NCI.future_series}_7_days_old", datetime_now - 7)
```

Comment:
Future and old_series probably need if guarding here as well? Currently they'd create an invalidly named directory ("mgm_cnf__6_days_old"). Not exactly a problem but also a bit pointless.